core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -29,20 +29,11 @@
 {% endblock header_js %}
 
 {% block navbar_breadcrumb %}
-  <ul class="nav navbar-nav oppia-navbar-breadcrumb" ng-controller="LearnerViewInfo">
-    <li>
-      <span class="oppia-navbar-breadcrumb-separator"></span>
-      <h1 class="oppia-exploration-h1"><span class="protractor-test-exploration-header oppia-exploration-header" itemprop="description">{{exploration_title}}</span></h1>
-    </li>
-    <li ng-click="showInformationCard()" uib-tooltip="<['I18N_PLAYER_INFO_TOOLTIP' | translate]>" tooltip-placement="bottom" style="cursor: pointer;" class="oppia-exploration-info-icon protractor-test-exploration-info-icon" tabindex="0">
-      <i class="material-icons oppia-navbar-breadcrumb-icon" style="font-size: 20px;">&#xE88E;</i>
-      <span class="oppia-icon-accessibility-label"><['I18N_PLAYER_INFO_TOOLTIP' | translate]></span>
-    </li>
-  </ul>
+  <learner-view-info exploration-title="{{exploration_title}}"></learner-view-info>
```

Comment:
Shouldn't this be `$ctrl.explorationTitle`?

Reply:
This is not AngularJS rendering. This is Jinja rendering, AngularJS renderings have `<[]>`. Also exploration_player is still not a directive template so we can't have `$ctrl` there. But again it is Jinja.

Does that make sense?

Reply:
/cc @Jamesjay4199 @vojtechjelinek as an aside, do you know what the plan is for getting rid of the Jinja here? I'm a bit concerned that it's starting to mix a bit too much with the Angular.

(@YashJipkate this doesn't block merge, but I want to make sure it's on their radar.)

Reply:
Yes, thanks for the explanation!